example/tests/Provider/PactVerifyTest.php

```diff
@@ -2,11 +2,11 @@
 
 namespace Provider;
 
-use GuzzleHttp\Client;
-use PhpPact\PactBrokerConnector;
-use PhpPact\PactUriOptions;
-use PhpPact\PactVerifier;
-use PhpPact\PactVerifierConfig;
+use GuzzleHttp\Psr7\Uri;
+use PhpPact\Broker\Service\BrokerHttpService;
+use PhpPact\Http\GuzzleClient;
```

Comment:
This project wraps the GuzzleClient?

Reply:
I made an interface to wrap the Http Client. That way we can swap from Guzzle to something else relatively easily in the future.